apps/judicial-system/api/src/app/modules/court/court.service.ts

```diff
@@ -1,10 +1,14 @@
+import { formatISO } from 'date-fns' // eslint-disable-line no-restricted-imports
```

Comment:
Can you write `import formatISO from 'date-fns/formatISO'` and remove the eslint thing?

Reply:
No, that results in failing tests and I don't know a way to fix that.